src/platform/platform_internal.h

```diff
@@ -917,6 +917,13 @@ typedef struct CXPLAT_SOCKET {
 
 } CXPLAT_SOCKET;
 
+typedef struct CXPLAT_REGISTERED_BUFFER_POOL {
+    void *Ring;
+    uint8_t *Buffers;
```

Comment:
nit, style puts the `*` with the type
```suggestion
    void* Ring;
    uint8_t* Buffers;
```

Reply:
Yeah, I will have to do one sweep for these after, because they keep creeping in.

Reply:
Swept. There were a lot...